src/plugins/kibana/public/management/app.html

```diff
@@ -10,9 +10,10 @@
       <li
         ng-if="sectionName"
         ng-repeat="item in section.items.inOrder"
-        ng-class="item.class">
+        ng-class="item.class"
+        tooltip="{{::item.tooltip}}" tooltip-placement="bottom" tooltip-popup-delay="400" tooltip-append-to-body="1">
```

Comment:
This ending bracket should go on its own line (just updated the styleguide re this):

```html
<li
        ng-if="sectionName"
        ng-repeat="item in section.items.inOrder"
        ng-class="item.class"
        tooltip="{{::item.tooltip}}" tooltip-placement="bottom" tooltip-popup-delay="400" tooltip-append-to-body="1"
>
```